modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -34,6 +34,10 @@ AssetListManagementToolbarDisplayContext assetListManagementToolbarDisplayContex
 </portlet:actionURL>
 
 <aui:form action="<%= deleteAssetListEntryURL %>" cssClass="container-fluid-1280" name="fm">
+	<liferay-ui:breadcrumb
```

Comment:
With this, you add Asset Library > asset name

It lacks of the
Asset Library > asset name > _Home_